json/Name_Chip_ActiveName.txt

```diff
@@ -2342,72 +2342,72 @@
 	{
 		"assign": "51410",
 		"jp_text": "急流矢１",
-		"tr_text": ""
+		"tr_text": "White-Water Arrows 1"
```

Comment:
As in white-water rapids, which seems to be the metaphor they're using for this PA.